src/Env.php

```diff
@@ -37,7 +37,7 @@ public static function get($name)
     {
         if (self::$options & self::USE_ENV_ARRAY) {
             $value = isset($_ENV[$name]) ? $_ENV[$name] : false;
-        } else {
+        } elseif(($value = getenv($name, true)) === false) {
             $value = getenv($name);
```

Comment:
The correct if flow should be:

```php
} elseif (self::$options & self::LOCAL_ONLY) {
    $value = getenv($name, true);
} else {
    $value = getenv($name);
}
```

Reply:
Hi Oscar, excellent solution to use a new option Env::LOCAL_ONLY !
I hope you can add it soon, so i could immediately start using it ..

Reply:
Would you like to do these changes? If you cannot, I can discard your pull request and do them.

Reply:
I updated the pull request, i thought it was better to have a fallback solution on global env values.
So I changed LOCAL_ONLY to LOCAL_FIRST.
Try to see if it's better now ..